lib/components/map_widget.dart

```diff
@@ -138,8 +137,6 @@ class MapWidgetState extends State<MapWidget> {
       _lineRoute = lineRoute;
       _linePassedRoute = linePassedRoute;
     });
-
-    if (!widget.isStatic) initUpdateRouteTimer();
```

Comment:
Ok so, you have deleted the initUpdateRouteTimer call, but you have not added a call at any other place.
Thus our routes will not be updating anymore, making "navigation" impossible

Reply:
Actually tried it out, routes no longer update (how could they?) :D

Reply:
you are right, dunno why this was deleted..